substrate/client/authority-discovery/src/worker.rs

```diff
@@ -418,7 +418,7 @@ where
 
 		debug!(
 			target: LOG_TARGET,
-			"Publishing authority DHT record peer_id='{local_peer_id}' addresses='{addresses:?}'",
+			"Authority DHT record peer_id='{local_peer_id}' with addresses='{addresses:?}'",
```

Comment:
Actually, I put `Publishing` in the log message on purpose, to make it clear that we are publishing local authority-discovery record, because I was confused by the logs :D

If you want to highlight that we don't publish empty lists, may be only print the message if the list is not empty?

Reply:
I've reverted to the original log and only print it on some addresses